### ISSUE TITLE
Shorten CTA Button

### ISSUE THREAD

**Issue by abiddiscombe** [abiddiscombe/notedeck#72]
Remove the "New" from the "New Button" text. The PlusIcon should have a "New" aria-label attribute to account for this.
